packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -348,6 +348,7 @@ export class SwapQuoter {
         if (calcOpts.rfqt !== undefined) {
             calcOpts.rfqt.quoteRequestor = new QuoteRequestor(
                 rfqtOptions ? rfqtOptions.makerAssetOfferings || {} : {},
+                rfqtOptions ? (rfqtOptions.altRfqCreds ? rfqtOptions.altRfqCreds : undefined) : undefined,
```

Comment:
can this just be `rfqtOptions ? rfqtOptions.altRfqCreds : undefined`?

Reply:
yea, that works, bc rfqtOptions.altRfqCreds can just be undefined